repair/featurize/constraintfeat.py

```diff
@@ -11,30 +12,39 @@
 # used for detecting violations in pos_values have a reference to only
 # one relation's (e.g. t1) attribute on one side and a fixed constant
 # value on the other side of the comparison.
-unary_template = Template('SELECT _vid_, val_id, count(*) violations ' \
-                           'FROM $init_table as t1, $pos_values as t2 ' \
-                           'WHERE t1._tid_ = t2._tid_ AND t2.attribute = \'$rv_attr\' '\
-                             'AND $orig_predicates AND t2.rv_val $operation $rv_val '\
-                           'GROUP BY _vid_, val_id')
+unary_template = Template('SELECT _vid_, val_id, count(*) violations '
```

Comment:
Multiline string here and below.

Reply:
won't change now for the same reason above